providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/ConfigCucumberTest.java

```diff
@@ -16,7 +16,8 @@
 @Order(value = Integer.MAX_VALUE)
 @Suite
 @IncludeEngines("cucumber")
-@SelectFile("test-harness/gherkin/config.feature")
+// @SelectFile("test-harness/gherkin/config.feature")
+@SelectDirectories("test-harness/gherkin")
```

Comment:
this file is just a unit test based on gherkin tests, and not a e2e test. we purposely selected only the config file. the execution will fail, as it will not be able to find all the steps